UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/FunctionalTests/POSAIOValidFunctionalScenarios.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Net;
+using FluentAssertions;
+using NUnit.Framework;
+using UKHO.PeriodicOutputService.API.FunctionalTests.Helpers;
+using static UKHO.PeriodicOutputService.API.FunctionalTests.Helpers.TestConfiguration;
+
+namespace UKHO.PeriodicOutputService.API.FunctionalTests.FunctionalTests
+{
+    [Category("POSAIOValidFunctionalScenarios")]
+    public class POSAIOValidFunctionalScenarios
+    {
+        private static readonly POSWebJobApiConfiguration posWebJob = new TestConfiguration().POSWebJobConfig;
+        private static readonly POSFileDetails posDetails = new TestConfiguration().posFileDetails;
+        private static readonly FSSApiConfiguration FSSAuth = new TestConfiguration().FssConfig;
+        private string DownloadedFolderPath { get; set; }
```

Comment:
same as before on common code

Reply:
HI Ray, These has been separated and added in a new class. Thanks